### PR TITLE
Linted whole codebase to standard

### PR DESCRIPTION
**Pull request by HugoGiraudel**

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -1,234 +1,232 @@
...
 
-      it('should detect it with multiline Value', function(){
-        var context = parser.contextParser(getContent('variableMultilineValue.test.scss'));
+      it('should detect it with multiline Value', function () {
+        var context = parser.contextParser(getContent('variableMultilineValue.test.scss'))
         assert.deepEqual(context, {
           type: 'variable',
           name: 'map',
-          value: '(\n  \"a\": \"b\",\n  \"c\": \"\"\n)',
+          value: '(\n  "a": "b",\n  "c": ""\n)',
```
Comment (pascalduez) [inline test/test.js]: ⚠️

File: index.js [hunk 2]
```diff
@@ -12,197 +12,208 @@ var CDocParser = require('cdocparser');
...
+    return codeStart + context.code.length + 1 // Add closing brace!
   }
-};
+}
 
 /**
  * SCSS Context Parser
  */
 var scssContextParser = (function () {
-    var ctxRegEx = /^(@|%|\$)([\w-_]+)*(?:\s+([\w-_]+)|[\s\S]*?\:([\s\S]*?)(?:\s!(\w+))?\;)?/;
```
Comment (pascalduez) [inline index.js]: ⚠️

File: test/test.js (hunk 1 above)
Comment (HugoGiraudel) [inline test/test.js]: There is no need to escape these characters (linter says). I think we used to use double quotes a while ago, hence the escape sequences.

File: index.js (hunk 2 above)
Comment (HugoGiraudel) [inline index.js]: There is no need to escape these characters.

File: test/test.js (hunk 1 above)
Comment (pascalduez) [inline test/test.js]: Make sense.

**Pull request merged**